Build/scripts/init_build.ps1

```diff
@@ -46,7 +46,7 @@ $remote = $remote.replace("mshttps", "https")
 # Get the pushId and push date time to use that for build number and build date time
 $uri = ("{0}/commits/{1}?api-version=1.0" -f $remote, $commitHash)
```

Comment:
should this be $CommitHashs? Also, i dont like that this file has variables with inconsistent casing but i guess its fine for now- would probably be better to just have camelCased variables